cmd/http/headers.go

```diff
@@ -57,9 +57,9 @@ const (
 	AmzStorageClass = "x-amz-storage-class"
 
 	// S3 object tagging
-	AmzObjectTagging = "X-Amz-Tagging"
-	AmzTagCount      = "X-Amz-Tag-Count"
-	AmzTagDirective  = "X-Amz-Tagging-Directive"
+	AmzObjectTagging = "x-amz-tagging"
+	AmzTagCount      = "x-amz-tag-count"
+	AmzTagDirective  = "x-amz-tagging-directive"
```

Comment:
not required to lower case the header - http.Header's Get() method is case insensitive.

Reply:
done